recipes/lrez/build.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
-export HTSLIB_ROOT=${PREFIX}/lib
-rm -rf build
-mkdir build
-cd build
-cmake ..
-make -j2
-cd ..
 
-mkdir -p $PREFIX/bin/
 
+export C_INCLUDE_PATH=${PREFIX}/include
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
+export LD_LIBRARY_PATH=${PREFIX}/lib
+export LIBRARY_PATH=${PREFIX}/lib
+
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="-lz -lm -lc -lboost_iostreams -L$PREFIX/lib"
```

Comment:
```suggestion
export LDFLAGS="${LDFLAGS} -lz -lm -lc -lboost_iostreams"
```
Ideally, the `-l` flags are set via `LIBS` instead of `LDFLAGS`.